aten/src/ATen/native/Constraints.cpp

```diff
@@ -84,7 +84,7 @@ Tensor _make_dep_token_cpu(
     c10::optional<bool> pin_memory_opt,
     c10::optional<c10::MemoryFormat> memory_format_opt) {
   return at::empty(
-      {}, dtype_opt, layout_opt, device_opt, pin_memory_opt, memory_format_opt);
+      0, dtype_opt, layout_opt, device_opt, pin_memory_opt, memory_format_opt);
```

Comment:
what did you need this ?

Reply:
Doing at.empty({}) seems to construct something where tensor.numel() != 0, so I changed it to be 0 so that tensor.numel() == 0. This function is not used anywhere rn.